src/helper/helper.js

```diff
@@ -0,0 +1,10 @@
+import jwt from "jsonwebtoken";
+
+const generateToken = (user) => {
+  const accessToken = jwt.sign(user, process.env.ACCESS_TOKEN_SECRET);
```

Comment:
Make the jwt's payload (i.e parameter user in your case) a json format.

Reply:
I think that is a way declare an argument but you can suggest with a particular case.